### DISCUSSION TITLE
Why getIndicesFromDefaultFilters returns an array with zeroth index when expected is empty? #17

### DISCUSSION THREAD

**Discussion by pranavirabatti** [Sunbird-Ed/Community#18] [Bugs] [answered]
Why select the first element from the dropdown even when you deselect all the options from the dropdown?

![172377237-5c9629a5-1f59-4c76-8aff-fb620b66cc39](https://user-images.githubusercontent.com/22250006/172542568-0d1a4179-7d56-45b8-8eb1-6f06b99338c3.png)

Expected is:

se_gradeLevels: [ ]
se_mediums: [ ]

@sharathkashyap, @luckynvdu @Krishnaj20

cc: @Renuka-S @komalm

**Comment by rajeevsathish** [comment, marked as answer]
@pranavirabatti which version of sunbird you are using as this issue has been fixed in release-4.9.0.

cc: @luckynvdu @sharathkashyap

**Reply by pranavirabatti** [reply]
@rajeevsathish SunbirdEd portal version 4.4.1
What is the workaround for this now in order to make it work in the older version?

**Reply by pranavirabatti** [reply]
Any update on this?

**Reply by rajeevsathish** [reply]
@pranavirabatti the Issue is fixed in the 4.9.0 release so you can use the release to get the issue fixed

**Reply by sharathkashyap** [reply]
@pranavirabatti Feel free to contribute the fix from your end for 4.4.1 release. We can provide the PR where the issue got fixed in later release.
@rajeevsathish Kindly provide the reference PR so the issue can get resolved in earlier version as well.

cc: @luckynvdu

**Reply by pranavirabatti** [reply]
@sharathkashyap sure will do it.

**Reply by rajeevsathish** [reply]
@pranavirabatti please refer this PR to fix it this issue in the older release

https://github.com/Sunbird-Ed/SunbirdEd-portal/commit/775d59e8abbb87309adfc5461d44e75507bd74a0

cc: @sharathkashyap

**Reply by pranavirabatti** [reply]
> @pranavirabatti Feel free to contribute the fix from your end for 4.4.1 release. We can provide the PR where the issue got fixed in later release. @rajeevsathish Kindly provide the reference PR so the issue can get resolved in earlier version as well.
> 
> cc: @luckynvdu

https://github.com/Sunbird-Ed/SunbirdEd-portal/pull/8137
